cpp/src/barretenberg/proof_system/verification_key/verification_key.test.cpp

```diff
@@ -1,43 +1,153 @@
 #include "barretenberg/common/test.hpp"
 #include "barretenberg/common/streams.hpp"
+#include "barretenberg/numeric/random/engine.hpp"
 #include "verification_key.hpp"
 
+namespace {
+auto& engine = numeric::random::get_debug_engine();
+}
+
 using namespace barretenberg;
 using namespace bonk;
 
+/**
```

Comment:
These tests are great! We should also test that `compress_native` in `cpp/src/barretenberg/proof_system/verification_key/verification_key.cpp` gives the same result as the compress_native in `cpp/src/barretenberg/stdlib/recursion/verification_key/verification_key.hpp`.

Reply:
I've add a test. That should suffice for now.